code/game/objects/items/devices/defib.dm

```diff
@@ -262,10 +262,12 @@
 
 //Checks for various conditions to see if the mob is revivable
 /obj/item/weapon/shockpaddles/proc/can_defib(mob/living/carbon/human/H) //This is checked before doing the defib operation
-	if((H.species.flags & NO_SCAN))
+	if((H.species.flags & NO_SCAN &! H.species.reagent_tag & IS_VOX))
 		return "buzzes, \"Unrecogized physiology. Operation aborted.\""
 	else if(H.isSynthetic() && !use_on_synthetic)
 		return "buzzes, \"Synthetic Body. Operation aborted.\""
+	else if(H.species.reagent_tag & IS_VOX && combat == 0)
+		return "buzzes, \"Voltage rating insufficient. Operation aborted.\""
```

Comment:
The two lines above should be moved **into** the organic check below. Vox IIRC count as organics, so if someone decides to make an FBP vox for whatever reason, you've effectively made it impossible to defib them.

Reply:
IIRC, that is, there's no combat mode jumper cables that the RD has in his locker.

Reply:
This is intended, but can't I just nest an if statement under it to check for isSynthetic() to negate the combat variable requirement, or add a multitool interaction to jumper cables to enable combat mode?